TesseractOCR/G8Tesseract.mm

```diff
@@ -644,6 +645,49 @@ - (NSString *)recognizedHOCRForPageNumber:(int)pageNumber {
     return nil;
 }
 
+- (NSData *)recognizedPDFForImages:(NSArray*)images {
```

Comment:
The advantage is that everything has been already implemented. I do not need to study PDF file format, it's structure and so on to use tesseract API.
But we should think about implementing this using Apple API, purhaps. Let me dig deeper into the question.